blocks/property-search-bar/common-function.js

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable no-param-reassign, no-plusplus, no-mixed-operators, no-unused-expressions, no-nested-ternary, eqeqeq, max-len */
+export const TOP_LEVEL_FILTERS = {
+  Price: { label: 'price', type: 'range' },
+  MinBedroomsTotal: { label: 'beds', type: 'select' },
+  MinBathroomsTotal: { label: 'baths', type: 'select' },
+  LivingArea: { label: 'square feet', type: 'range' },
+};
+export const EXTRA_FILTERS = {
+  PropertyType: { label: 'property type', type: 'property' },
+  Features: { label: 'keyword search', type: 'keywords-search' },
+  MatchAnyFeatures: { label: 'Match', type: 'child' },
+  YearBuilt: { label: 'year built', type: 'range' },
+  NewListing: { label: 'new listings', type: 'toggle' },
+  RecentPriceChange: { label: 'recent price change', type: 'toggle' },
+  OpenHouses: { label: 'open houses', type: 'open-houses' },
+  Luxury: { label: 'luxury', type: 'toggle' },
+  FeaturedCompany: { label: 'berkshire hathaway homeServices listings only', type: 'toggle' },
+};
+
+const SQUARE_FEET = [
+  { value: '500', label: '500 Sq Ft' },
+  { value: '750', label: '750 Sq Ft' },
+  { value: '1000', label: '1,000 Sq Ft' },
+  { value: '1250', label: '1,250  Sq Ft' },
+  { value: '1500', label: '1,500 Sq Ft' },
+  { value: '1750', label: '1,750 Sq Ft' },
+  { value: '2000', label: '2,000 Sq Ft' },
+  { value: '2250', label: '2,250 Sq Ft' },
+  { value: '2500', label: '2,500 Sq Ft' },
+  { value: '2750', label: '2,750 Sq Ft' },
+  { value: '3000', label: '3,000 Sq Ft' },
+  { value: '3500', label: '3,500 Sq Ft' },
+  { value: '4000', label: '4,000 Sq Ft' },
+  { value: '5000', label: '5,000 Sq Ft' },
+  { value: '7500', label: '7,500 Sq Ft' },
+];
+
+const YEAR_BUILT = [
+  { value: '1900', label: '1900' },
+  { value: '1920', label: '1920' },
+  { value: '1940', label: '1940' },
+  { value: '1950', label: '1950' },
+  { value: '1960', label: '1960' },
+  { value: '1970', label: '1970' },
+  { value: '1980', label: '1980' },
+  { value: '1990', label: '1990' },
+  { value: '1995', label: '1995' },
+  { value: '2000', label: '2000' },
+  { value: '2005', label: '2005' },
+  { value: '2014', label: '2014' },
+  { value: '2015', label: '2015' },
+  { value: '2016', label: '2016' },
+  { value: '2017', label: '2017' },
+  { value: '2018', label: '2018' },
+  { value: '2019', label: '2019' },
```

Comment:
This is just me - they stopped updating their list at 2019? What about houses build since then? Never show up in search? 😄

Reply:
:))